lib/cli.js

```diff
@@ -3,23 +3,62 @@
 const { checkNodeCompatibility } = require('../lib/util');
 checkNodeCompatibility();
 
-// proceed only if node version is good.
+// proceed iff the node version is good.
 const ora = require('ora');
 const { displayResults } = require('../lib/output/cli-output');
-const { processPolicies } = require('../lib/project/mulitple-projects');
+const { processPolicies } = require('../lib/project/multiple-projects');
 const { DEFAULT_SUPPORT_MESSAGE } = require('./output/messages');
 const { generateCsv } = require('../lib/output/csv-output');
+const parseDuration = require('parse-duration');
+/*
+ *
+ * convert a string representing a date into an actual date
+ * supports anything new Date(maybeDate) supports
+ *
+ * supports anything `parse-duration` supports:
+ *   -30 days === the date 30 days ago
+ */
```

Comment:
Any reason not to use jsdoc style comments?

Reply:
More of an inline note, rather than API docs. I would prefer to convert the project to TS for better API docs derived from types etc. (which I believe we intend to do once this lands)

Reply:
Fair.